fastify.js

```diff
@@ -44,6 +44,12 @@ const reqIdGenFactory = require('./lib/reqIdGenFactory')
 const getSecuredInitialConfig = require('./lib/initialConfigValidation')
 
 const DEFAULT_BODY_LIMIT = 1024 * 1024 // 1 MiB
+const DEFAULT_ENABLE_CASE_SENSITIVE = true
```

Comment:
I think that there is no need to duplicate this code twice, you can declare `defaultInitOptions` in `lib/initialConfigValidation` and export it here.
In this way, the code is divided into logical components, as we are trying to do with #1475.

Reply:
@delvedor > ok so to be sure that I understood, you suggest that I move all the default constants inside `lib/initialConfigValidation` where I should declare a const object like this:
```js
const defaultInitOptions = {
  bodyLimit: 1024 * 1024, // 1 MiB
  caseSensitive: true,
  ignoreTrailingSlash: false,
  maxParamLength: 100,
  onProtoPoisoning: 'error',
  pluginTimeout: 10000,
  requestIdHeader: 'request-id'
}
```
Then export it as `defaultInitOptions` in `fastify.js` and use these properties as a default fallback option. And in the meantime I change back the signature to `getSecuredInitialConfig(options)` because all the defaults will be in the same file as the one that will do the processing logic.

Did I understood correctly ? Is this what you had in mind ?

Reply:
Yes! 🚀

Reply:
Nice and clever :P
I will write this right now ^^